Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+pipeline {
```

Comment:
I don't know enough about `Jenkinsfile`s to judge if this is really correct, but no part of it seems unreasonable.